### PR TITLE
[Feature] Basic Chat Views

### PR DESCRIPTION
**Pull request by palatter**
## Purpose

* Title
* Number of participants
* Message thread
* Message bubble
* Message Input

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Presentation/SwiftUI/Chat/ChatView.swift [hunk 1]
```diff
@@ -6,9 +6,33 @@
...
     }
 }
+
+// struct ChatView_Previews: PreviewProvider {
+//    static var previews: some View {
+//        let compositeViewModelFactory
+//    let viewModel = ChatViewModel(
+//        ChatView(viewModel: <#T##ChatViewModel#>)
+//    }
+// }
```
Comment (jimchou-dev) [inline AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Presentation/SwiftUI/Chat/ChatView.swift:38]: nitpick: Remove?

File: AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Presentation/SwiftUI/Chat/ChatViewComponents/BubbleMessageView.swift [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+// Crashes?
+// struct BubbleMessageView_Previews: PreviewProvider {
+//    static var previews: some View {
+//        Text("Test")
+//        BubbleMessageView(message: "Hello World",
+//                          createdOn: Date(),
+//                          displayName: "John Smith",
+//                          isSelf: true)
+//    }
+// }
```
Comment (jimchou-dev) [inline AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Presentation/SwiftUI/Chat/ChatViewComponents/BubbleMessageView.swift:86]: nitpick: remove?

**Review by jimchou-dev — APPROVED** [review]

**Comment by jpeng-ms** [comment]
looks good, but in the log, I noticed this warning, is this happening because we hard-coded the chat messages for now? 
image

**Comment by palatter** [comment]
Oh that's because I've hardcoded an empty I'd value for now. That will be resolved with real data

**Review by jpeng-ms — APPROVED** [review]

**Pull request merged**